tensorboard/plugins/scalar/vz_line_chart/vz-line-chart.ts

```diff
@@ -335,18 +405,18 @@ class LineChart {
         [[this.yAxis, this.center], [null, this.xAxis]]);
   }
 
-  private buildPlot(xAccessor, xScale, yScale): Plottable.Component {
-    this.scalarAccessor = (d: ChartHelpers.ScalarDatum) => d.scalar;
+  private buildPlot(
+      xAccessor, yAccessor, xScale, yScale, tooltipColumns): Plottable.Component {
```

Comment:
Can we either remove the `yAccessor` parameter or change occurrences of `this.yValueAccessor` to `yAccessor` in the function body?